cfgov/jinja2/v1/_includes/templates/forms/accessibility-feedback-form.html

```diff
@@ -1,37 +1,124 @@
-{% import 'macros/util/form.html' as form %}
-{% import 'macros/util/common-form-fields.html' as form_field %}
-<section class="block
-                block__bg
-                block__border
-                block__flush-sides">
-    <form class="form-l js-validate-filters" id="accessibility-form">
-        <div class="form-l_col
-                    form-l_col-1
-                    block
-                    block__flush-top
-                    block__flush-bottom">
-          <h2>Form to file your accessibility complaint</h2>
-        </div>
-
-        {{ form.input('First Name', 'text', '1-2', true) }}
+<form class="js-validate-filters" id="accessibility-form">
+    <h2>Form to file your accessibility complaint</h2>
+    <section class="o-form-section">
+        {% do global_dict.update({
+            'input': {
+                'label':        'First name',
+                'type':         'text',
+                'size':         '1-2',
+                'required':     true,
+                'disabled':     false
+            }
+        }) %}
+        {% set value = global_dict.input %}
+        {% set prototype = true %}
+        {% include 'atoms/input.html' with context %}
 
-        {{ form.input('Last Name', 'text', '1-2', true) }}
+        {% do global_dict.update({
+            'input': {
+                'label':        'Last name',
+                'type':         'text',
+                'size':         '1-2',
+                'required':     true,
+                'disabled':     false
+            }
+        }) %}
+        {% set value = global_dict.input %}
+        {% set prototype = true %}
```

Comment:
Does `prototype` need to be set every time or could it just be done once at the top?